### PR TITLE
ndb cluster

### PR DESCRIPTION
**Pull request by Abhishekism9450**
NDB Cluster Resource.

### CODE REVIEW

File: examples/ndb/cluster/main.tf [hunk 1]
```diff
@@ -0,0 +1,45 @@
+terraform{
+    required_providers {
+        nutanix = {
+            source = "nutanix/nutanix"
+            version = "1.8.0-beta"
```
Comment (bhati-pradeep) [inline examples/ndb/cluster/main.tf]: 1.8.0

File: nutanix/resource_nutanix_ndb_cluster.go [hunk 2]
```diff
@@ -0,0 +1,611 @@
...
+
+	if err := d.Set("resource_config", flattenResourceConfig(resp.Resourceconfig)); err != nil {
+		return diag.FromErr(err)
+	}
+
+	if err := d.Set("management_server_info", resp.Managementserverinfo); err != nil {
+		return diag.FromErr(err)
+	}
+
+	if err := d.Set("entity_counts", resp.Entitycounts); err != nil {
```
Comment (bhati-pradeep) [inline nutanix/resource_nutanix_ndb_cluster.go]: can we follow naming like "EntityCounts" for all above as well

Comment (bhati-pradeep) [reply nutanix/resource_nutanix_ndb_cluster.go]: For better understanding of struct names

File: website/docs/r/ndb_cluster.html.markdown [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            ]
+        }
+    }
+```
+
+## Argument Reference
+
+* `name`: (Required) name of the cluster to be registered
+* `description`: (Optional) description of cluster
+* `cluster_ip`: (Required) Type the Prism Element IP address
```
Comment (bhati-pradeep) [inline website/docs/r/ndb_cluster.html.markdown]:
```suggestion
* `cluster_ip`: (Required)  Prism Element IP address
```

**Review by bhati-pradeep — CHANGES_REQUESTED** [review]

File: nutanix/resource_nutanix_ndb_cluster.go (hunk 2 above)
Comment (Abhishekism9450) [reply nutanix/resource_nutanix_ndb_cluster.go]: Done.

**Review by bhati-pradeep — APPROVED** [review]

**Review by abhimutant — APPROVED** [review]

**Pull request merged**